adapters/bidmachine/bidmachinetest/exemplary/rewarded_interstitial.json

```diff
@@ -0,0 +1,129 @@
+        "imp": [
+        {
+            "id": "test-imp-id",
+            "instl": 1,
+            "banner":
+            {
+                "battr": [1],
+                "mimes": [
+                    "image/jpeg",
+                    "image/jpg",
+                    "image/gif",
+                    "image/png"
+                ],
+                "w": 320,
+                "h": 480
+            },
+            "ext":
+            {
+                "prebid":
+                {
+                    "is_rewarded_inventory": 1
+                },
+                "bidder":
+                {
+                    "host": "api-us",
+                    "path": "auction/rtb/v2",
+                    "seller_id": "1"
+                }
+            }
+        }]
+    },
+    "httpCalls": [
+    {
+        "expectedRequest":
+        {
+            "uri": "https://api-us.bidmachine.io/auction/rtb/v2/1",
+            "headers":
+            {
+                "Content-Type": [
+                    "application/json"
+                ],
+                "Accept": [
+                    "application/json"
+                ],
+                "X-Openrtb-Version": [
+                    "2.5"
+                ]
+            },
+            "body":
+            {
+                "id": "test-request-id",
+                "imp": [
+                {
+                    "id": "test-imp-id",
+                    "instl": 1,
+                    "banner":
+                    {
+                        "battr": [1, 16],
+                        "mimes": [
+                            "image/jpeg",
+                            "image/jpg",
+                            "image/gif",
+                            "image/png"
+                        ],
+                        "w": 320,
+                        "h": 480
+                    },
+                    "ext":
+                    {
+                        "prebid":
+                        {
+                            "is_rewarded_inventory": 1
+                        },
+                        "bidder":
+                        {
+                            "host": "api-us",
+                            "path": "auction/rtb/v2",
+                            "seller_id": "1"
+                        }
+                    }
+                }]
+            }
+        },
+        "mockResponse":
+        {
+            "status": 200,
+            "body":
+            {
+                "id": "test-request-id",
+                "seatbid": [
+                {
+                    "seat": "seat-id",
+                    "bid": [
+                    {
+                        "id": "test-bid-id",
+                        "impid": "test-imp-id",
+                        "price": 100,
+                        "adm": "test-adm",
+                        "cid": "test-cid",
+                        "crid": "test-crid"
+                    }],
+                    "seat": "bidmachine"
```

Comment:
The tests have two definitions for `"seat"`. One as `"seat-id"` and the other as `"bidmachine"`. I don't think the value actually matters for the purpose of these tests. Consider removing `"seat": "seat-id",` and leaving in ` "seat": "bidmachine"`.